source/app/requesthandlers/scheduler_setup_handler.py

```diff
@@ -222,22 +270,52 @@ def handle_request(self):
         finally:
             self._logger.flush()
 
-    def _update_settings(self):
+    def is_valid_org_id(self, org_id):
+        """
+        Verifies if the ou_id param is a valid ou_id format. https://docs.aws.amazon.com/organizations/latest/APIReference/API_Organization.html
+        :return: boolean
+        """
+        try:
+            return re.fullmatch("^o-[a-z0-9]{10,32}$", org_id)
+        except Exception as error:
+            raise error
```

Comment:
excepting then immediately raising is redundant

Reply:
It may be redundant from a behavior perspective, but it does makes the ability of the code to throw an exception explicit rather than implicit, which I would personally consider to generally be a positive (whether or not that applies here specifically is up for some debate though).